source/robot_model/include/robot_model/Model.hpp

```diff
@@ -141,6 +155,43 @@ class Model {
   StateRepresentation::Jacobian compute_jacobian(const StateRepresentation::JointState& joint_state,
                                                  const std::string& frame_name = "");
 
+  /**
+   * @brief Compute the Inertia matrix from a given joint positions
+   * @param joint_positions containing the joint positions values of the robot
+   * @return the inertia matrix
+   */
+  Eigen::MatrixXd compute_inertia_matrix(const StateRepresentation::JointPositions& joint_positions);
+
+  /**
+   * @brief Compute the Inertia torques, i.e the inertia matrix multiplied by the joint accelerations. Joint positions
+   * are needed as well for computations of the inertia matrix
+   * @param joint_state containing the joint positions and accelerations values of the robot
+   * @return the inertia matrix
+   */
+  StateRepresentation::JointTorques compute_inertia_torques(const StateRepresentation::JointState& joint_state);
+
+  /**
+   * @brief Compute the Coriolis matrix from a given joint state
+   * @param joint_state containing the joint positions & velocities values of the robot
+   * @return the Coriolis matrix
+   */
+  Eigen::MatrixXd compute_coriolis_matrix(const StateRepresentation::JointState& joint_state);
+
+  /**
+   * @brief Compute the Coriolis forces, i.e. the coriolis matrix multiplied by the joint velocities and express the
+   * result as a JointTorques
+   * @param joint_state containing the joint positions & velocities values of the robot
+   * @return the Coriolis forces as a JointTorques
+   */
+  StateRepresentation::JointTorques compute_coriolis_forces(const StateRepresentation::JointState& joint_state);
```

Comment:
here you use the term "_forces" in generalised coordinates, but in the other methods you use "_torques". I would prefer this to be consistent, and of the two choices "_torques" is more obviously referring to joint space

Reply:
Right. I was actually looking at dynamic lectures and most of the time this vector is called `coriolis_forces` but I was also thinking it would be better to have consistency.